C-Plus-Plus/graphs/Prims_Algo_PriorityQueue.cpp

```diff
@@ -0,0 +1,158 @@
+//Prims Algorithm to find Minimum Spanning of a Graph using Priority Queue 
+
+#pragma GCC optimize("O3")
+#pragma GCC optimize("Ofast")
+#pragma GCC optimize("unroll-loops")
+#pragma GCC optimize("no-stack-protector")
+#pragma GCC optimize("fast-math")
+#pragma GCC optimize("trapv")
+#pragma GCC target("sse4")
+

```

Comment:
- Remove these lines
- Add Multi-Line Comments 
- Add a beginner-friendly description at the top